composer.json

```diff
@@ -59,7 +59,9 @@
         "brainmaestro/composer-git-hooks": "^2.8.5",
         "phpspec/prophecy-phpunit": "^2.0.2",
         "phpstan/phpstan": "^1.10.14",
-        "phpunit/phpunit": "^9.6.7"
+        "phpunit/php-code-coverage": "^10",
+        "phpunit/php-file-iterator": "^4.0",
```

Comment:
I forget: Is this here because we had to pin to a specific version that would be installable on all supported versions of PHP? Because there's no way we're actually using it directly. If so, cool. If not, let's remove it.

Reply:
We are using this in our static analysis tests , check here https://github.com/acquia/orca/actions/runs/9027070083/job/24805359219?pr=420#step:9:161. Marking this as resolved and merging